### PR TITLE
LPS-189476 Copy tag ratings and categories when coping files and folders

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
Resent after changes asked: brianchandotcom#138448 (comment)

To test it :
For all: Enable FF: LPS-182512

1. Go to global site
2. On this create a category and a tag
3. Create a Document with created category and tag
4. Go to document > dropdown > copy to
5. copy to Liferay site (home)
6. Go to Liferay site > Documents and Media
7. check document copied from global (edit) has the tag and the category

other test

1. Create an Asset Library
2. Connect to Liferay Site
3. On the asset library create a category and a tag
4. Create a Document with created category and tag
5. Go to document > dropdown > copy to
6. copy to Liferay site (home)
7. Go to Liferay site > Documents and Media
8. check document copied from asset library has the tag and the category

note : if copy between sites not related, only common categories and tags (from global or an common ancestor) is copied

This should work with files and folders

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:lima

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCActionCommand.java [hunk 1]
```diff
@@ -60,14 +61,10 @@ protected void doProcessAction(
...
-	private void _checkDestinationRepository(long repositoryId)
-		throws PortalException {
-
-		Group group = _groupLocalService.fetchGroup(repositoryId);
-
+	private void _checkDestinationGroup(Group group) throws PortalException {
 		if ((group != null) && group.isStaged() && !group.isStagingGroup()) {
 			throw new PortalException(
-				"cannot-copy-file-entries-to-the-live-version-of-a-group");
+				"cannot-copy-folders-to-the-live-version-of-a-group");
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCActionCommand.java:67]: Is this key change correct? Looks like a typo to me.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCActionCommand.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCActionCommand.java:67]: oh! 
Thanks!

I will rechange it!!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:lima

**Review by adolfopa — APPROVED** [review]

**Comment by austinchiang** [comment]
ci:forward:force

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1257401_2513

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1317401_8998

**Comment by austinchiang** [comment]
ci:forward:force

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:forward:force

**Pull request closed without merging**